spec/menu_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'menu'
+
+describe Menu do
+  subject(:menu) { described_class.new }
+
+  describe '#options' do
+    it 'shows menu options when called' do
+      expect(menu.options).to eq({"beef-chuck brisket sub"=>4.5, "pastrami salami sub"=>5.0})
+    end
+  end
+
+  describe '#add_item' do
+    it 'allows restaurant owner to add items to menu' do
+      menu.add_item('chicken duck', 5.50)
+      expect(menu.options).to eq({"beef-chuck brisket sub"=>4.5, "pastrami salami sub"=>5.0, "chicken duck"=>5.5})
```

Comment:
Metrics/LineLength: Line is too long. [114/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.